workflow/rules/sbs.smk

```diff
@@ -49,9 +49,11 @@ rule compute_standard_deviation:
 # Find local maxima of SBS reads across cycles
 rule find_peaks:
     input:
-        SBS_OUTPUTS["compute_standard_deviation"],
+        SBS_OUTPUTS["compute_standard_deviation"] if config["sbs"]["spot_detection_method"] == "standard" else SBS_OUTPUTS["align_sbs"],
```

Comment:
This is a cool way to skip the steps that aren't needed for other spot detection methods 🔥